lib/class-wp-rest-widget-types-controller.php

```diff
@@ -472,8 +472,10 @@ public function encode_form_data( $request ) {
 				)
 			),
 			'instance' => array(
-				'encoded' => base64_encode( $serialized_instance ),
-				'hash'    => wp_hash( $serialized_instance ),
+				'encoded'                       => base64_encode( $serialized_instance ),
+				'hash'                          => wp_hash( $serialized_instance ),
+				'title'                         => $request['instance']['title'],
+				'is_widget_customizer_js_value' => $request['instance']['is_widget_customizer_js_value'],
```

Comment:
Hm, I don't think these REST API endpoints should know anything about Customizer stuff (including `is_widget_customizer_js_value`). Is it to fix the issue if you add a new Legacy Widget?